### DISCUSSION TITLE
Any ETA for v5?

### DISCUSSION THREAD

**Discussion by Zerebokep** [ant-design/ant-design#36280] [General]
My team planned to use ant design for a large scale project, unfortunately the current ConfigProvider does not satisfy our needs for dynamic theming, but it seems v5 solves this problem completely, hence the question when that new version or a alpha will be available.

**Comment by shifenhutu** [comment]
there is have an  experimental version ,   https://www.npmjs.com/package/antd/v/5.0.0-experimental.0

i dont know the release version  release date
